core/templates/pages/topic-editor-page/subtopic-editor/subtopic-editor-tab.component.spec.ts

```diff
@@ -58,7 +58,7 @@ describe('Subtopic editor tab', function() {
     $location = $injector.get('$location');
 
     var MockQuestionBackendApiService = {
-      fetchTotalQuestionCountForSkillIds: () => Promise.resolve(2)
+      fetchTotalQuestionCountForSkillIds: async() => Promise.resolve(2)
```

Comment:
Ditto.

Reply:
I was adviced that I can ignore adding Async keyword at the end of function name as I am working on a spec file.